todos.js

```diff
@@ -1,5 +1,21 @@
-const el = document.querySelector(".jsonSection");
+const el = document.querySelector(".json-section");
 
-fetch("https://jsonplaceholder.typicode.com/todos/1")
+fetch("https://jsonplaceholder.typicode.com/todos")
   .then(response => response.json())
-  .then(json => (el.innerHTML = json.title));
+  .then(json => renderJson(json));
+
+const renderJson = json => {
+  el.innerHTML = json.title;
+  const userTasks = json
+    .map(
+      x => `<ul class="list">
```

Comment:
Put the `ul` onto a new line and format the template string as it would look in the html